cypress/e2e/signIn.cy.js

```diff
@@ -1,11 +1,45 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  const username = 'tomsmith';
+  const password = 'SuperSecretPassword!';
+
   beforeEach(() => {
+    cy.visit('/login');
+  });
 
+  it('should Login with valid creds', () => {
+    /*  cy.get('#username').type(username)
+        cy.get('#password').type(password)
+        cy.get('.fa.fa-2x.fa-sign-in').click()  */
+    cy.RegValidUser(username, password);
```

Comment:
```suggestion
    cy.loginUser(username, password);
```
this is not registration. This function logs in the user